tests/constants.py

```diff
@@ -42,27 +42,31 @@
         SHA = ""
 
 CACHED_CONFIG_FILE = None
-SERVER_CONFIG_FILE = "mswms_settings.py"
+MSWMS_SERVER_CONFIG_FILE = "mswms_settings.py"
```

Comment:
```suggestion
MSWMS_CONFIG_FILE = "mswms_settings.py"
```

To apply the same naming scheme that is also already used for `MSCOLAB_CONFIG_FILE` below.

Reply:
that was the idea, we know that it is a server.